src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +57,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrinatbleString() {
+		return ADDRESS + toString();
```

Comment:
please follow coding standards. use four space for an indentation, not a tab character.